### PR TITLE
Fix sidebar updateTabs method

### PR DESCRIPTION
**Pull request by skjnldsv**
Needed for 20! :warning: 
Fix https://github.com/nextcloud/photos/issues/467

### CODE REVIEW

**Comment by skjnldsv** [comment]
> [@skjnldsv](/skjnldsv) [skjnldsv](/skjnldsv) changed the title ~improve handling of out of space errors for smb~ Fix sidebar updateTabs method [maintenant](#event-3854384860)

Go home Github, you're drunk!

File: apps/files/src/views/Sidebar.vue [hunk 1]
```diff
@@ -259,7 +259,7 @@ export default {
 
 					this.$nextTick(() => {
 						if (this.$refs.sidebar) {
-							this.$refs.sidebar.updateTabs()
+							this.$refs.tabs.updateTabs()
```
Comment (juliushaertl) [inline apps/files/src/views/Sidebar.vue:262]: Shouldn't it check for that ref then in the if statement above as well :wink:

Comment (skjnldsv) [reply apps/files/src/views/Sidebar.vue:262]: :roll_eyes:  :sweat_smile:

Comment (skjnldsv) [reply apps/files/src/views/Sidebar.vue:262]: done

**Review by rullzer — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Comment by skjnldsv** [comment]
@MorrisJobke that was on stable20

**Comment by skjnldsv** [comment]
master have it alongside other changes, I backported this only

**Comment by MorrisJobke** [comment]
🙈

**Pull request merged**